zuul.d/layout.yaml

```diff
@@ -6,7 +6,7 @@
     vars:
       tox_envlist: py27-ansible28-unit
       tox_environment:
-        PYTEST_REQPASS: 559
+        PYTEST_REQPASS: 472
```

Comment:
this is very weird thing to do

Reply:
I know but it worked fine so far. Even got a new PR from a new user which didn't ask anything about it, he just updated the number because he was adding a new test.

I bet you will see the real benefit as soon we will have something changes on Zuul nodes that triggers some tests to skip. We still have to re-enable podman ones, which are skipped now.